gitpod-io/website
# Align table styles with design system

## Description



Increases row size which is relatively large than headers.

<img width="784" alt="image" src="https://github.com/gitpod-io/website/assets/55068936/802490db-b4c1-418a-8674-8edf4957accd">

<img width="784" alt="image" src="https://github.com/gitpod-io/website/assets/55068936/3ed9a567-586a-4ffc-b3fa-754d179c7b04">



## Related Issue(s)



Fixes MKWEB-10

## How to test



- Open Preview 
-  Go to `/docs/references/gitpod-public-api#gitpod-public-api`
- See table :eyes:



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3719"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
